src/content/docs/tutorial-optimize-telemetry/detect-anomalies.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Detect ingest anomalies"
+metaDescription: Use your baseline report to detect ingest anomalies.
+redirects:
+---
+
+import omaoedgIngestTargetLine from 'images/oma-oe-dg_screenshot-full_ingest-target-line.webp'
+
+import omaoedgLookoutViewConsumingAccount from 'images/oma-oe-dg_screenshot-crop_lookout-view-consuming-account.webp'
+
+Once you've installed the baseline dashboard `link here`, you should keep an eye on it to detect any ingest anomalies as often as possible. While you have to wait until you have enough data to optimize your ingest, you can use your dashboard to find issues with your ingest much sooner. You can use the baseline dashbaord as is, or download optional dashboards to provide you even greater anomaly detection options.
+
+You should also set up alerts to help notify you of issues as they occur. You can use our [ingest alerts guide](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts) to create a large variety of alerts, but we recommend you set up these two alerts as a minimum:
+
+* A threshold alert to notify if you exceed monthly targets for data ingest beyond seasonal increases
+* An anomaly alert to notify you of a sudden sharp increase ingest data
+
+<Callout variant="tip">
+New to creating alerts with our platform? Check out our [alert creation and management tutorial](/docs/tutorial-create-alerts/create-new-relic-alerts/) to learn everything you need about creating New Relic alerts.
+</Callout>
+
+## Use the Lookout view to monitor anomalies [#lookout]
```

Comment:
Just to confirm, is it Lookout or Lookup?

Reply:
It's lookout as per an SME review!